packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.tsx

```diff
@@ -56,17 +56,6 @@ export const RadioGroup = compose<RadioGroupType>({
         </Slots.label>
       );
 
-      // Populate the buttonKeys array
-      if (children) {
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore - TODO, fix typing error
-        contextValue.values = React.Children.map(children, (child: React.ReactChild) => {
```

Comment:
It might be helpful to a reviewer to note this change in the PR description rather than having to read through the code and figure out why we're deleting this important bit of code.

I try to always note all changes in the PR description in the case that a reviewer is confused by a block of code, they can go to the description and figure out the change as a whole.

Reply:
Thanks for the feedback! Updated the PR description.